shell/initialize/index.js

```diff
@@ -254,76 +291,98 @@ async function createApp(ssrContext, config = {}) {
   //   await nuxt_plugin_portalvue_6babae27(app.context, inject);
   // }
 
+  // - no need for this gating....
```

Comment:
removing gates on process.client makes sense. for other conditions though it would be nice to know why it's here before removing